packages/components/src/form-toggle/style.scss

```diff
@@ -27,20 +49,28 @@ $toggle-border-width: 1px;
 		box-sizing: border-box;
 		top: $toggle-border-width * 3;
 		left: $toggle-border-width * 3;
-		width: $toggle-height - ($toggle-border-width * 6);
-		height: $toggle-height - ($toggle-border-width * 6);
+		width: $thumb-size;
+		height: $thumb-size;
 		border-radius: 50%;
-		transition: 0.1s transform ease;
+		transition:
+			$transition-duration transform ease,
+			$transition-duration background-color ease-out;
 		@include reduce-motion("transition");
 		background-color: $gray-900;
-		border: 5px solid $gray-900; // Has explicit border to act as a fill in Windows High Contrast Mode.
+
+		// Transparent border acts as a fill in Windows High Contrast Mode.
+		border: $thumb-size / 2 solid transparent;
```

Comment:
This appears to be throwing a deprecation warning when building the plugin locally. I ran out of time today to put up a quick fix, but just thought I'd mention it:

<img width="916" alt="image" src="https://github.com/WordPress/gutenberg/assets/14988353/f9da82cf-0f33-41ad-b9e7-af1397938217">

Reply:
I think you can do this, to enable the SASS:

`border: #{ $thumb-size / 2 } solid transparent;`

That will make the calculation in the sass part of things, instead of the CSS part of things.

Reply:
Oh wait, I might be wrong. The other option is to do `$thumb-size * 0.5` — I think it's because the slash is now also used for some grid CSS properties, so it's not good as a math operator anymore.

Reply:
Right! I'll spin up a PR to implement @jasmussen 's suggestion. Thank you!

Reply:
[The official recommendation](https://sass-lang.com/documentation/breaking-changes/slash-div/) is to `@use "sass:math"`. So in theory, this...

```scss
border: math.div($thumb-size, 2) solid transparent;
```